### PR TITLE
Added myvirus and supporting data

### PR DESCRIPTION
**Pull request by joackmak**

### CODE REVIEW

File: virus/Arcanarumenalis.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
+# Unearthing Arcana ruminalis: A Novel Virus Shaping the Bovine Gut Microbiome in Alberta Ecosystems"
+written by: [Makuach Joack](https://github.com/joackmak *optional link*)
```
Comment (ababaian) [inline virus/Arcanarumenalis.md]: Fix link to your account. Remove the optional linnk text.

File: virus/Arcanarumenalis.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+Our results contribute to a broader understanding of novel RNA-based entities, shedding light on their ecological roles and evolutionary pathways. This work underscores the importance of characterizing unconventional genomes to redefine the boundaries of viral taxonomy and their functional significance within host ecosystems.
+
+## Results
+
+### Ecology is defined as the interaction between the organisms and their environment, it may be biotic (consider other organisms as living components) and abiotic (the non-living homeostasis, such as climate, soil, and water). Computational analysis of genomic and environmental data sheds light on relationships between viral organisms and their environments in the context of digital ecology. Digital ecology utilizes bioinformatics tools to investigate the host−virus interaction of viruses like the Obelisk in multiple host species and environmental conditions in multiple ecosystems using the data acquired from sequencing technologies and other environmental datasets.
```
Comment (ababaian) [inline virus/Arcanarumenalis.md]: This is title, it won't look good if the paragraph is all in title-space. Make the title descriptive of your primary conclusion for that section.

File: virus/Arcanarumenalis.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+To address the hypothesis that Arcana ruminalis interacts with specific host species, particularly bovine species, I queried PebbleScout[4] using the nucleotide sequences of Arcana ruminalis. The analysis revealed that the first 100 cases consistently exhibited high raw scores, PBS scores, and percentage coverage across all SRA datasets derived from rumen microbiomes. In contrast, detection in datasets from non-rumen sources was significantly lower.
+
+This strong signal in rumen-specific data likely support the hypothesis that Arcana ruminalis is closely associated with bovine gut ecosystems, specifically the rumen. Its absence or low prevalence in other datasets reinforces its potential specialization or ecological dependence on the bovine gut microbiome. These findings hint to suggest that Arcana ruminalis may indeed play a functional role in the microbial dynamics of the rumen, warranting further investigation into its interactions and contributions within this environment.
+
+
+### Virus Genome {Q3}
```
Comment (ababaian) [inline virus/Arcanarumenalis.md]: Amend out the "Default Titles" from the document. Your descriptive titles are much better throughout.

File: virus/Arcanarumenalis.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+
+To annotate the genome of Arcana ruminalis, I began by running the sequence of the centroid through NCBI ORFfinder  to identify all potential open reading frames (ORFs). Each ORF was subsequently analyzed through BLASTp searches to determine homology and functional annotations for the predicted proteins. Using the results, I compiled a comprehensive genomic dataframe that integrated the ORF details with their corresponding BLASTp annotations. Finally, leveraging R, I created a detailed genome map to visually represent the annotated features and provide a clear overview of the genomic organization.
+![A. ruminalis Genome Map](img/Arcana_rumenalis/GenomeMap.png)
+
+
+Figure 1 : Annotation of Arcana rumenalis genome.
```
Comment (ababaian) [inline virus/Arcanarumenalis.md]: Figure Legend?

**Review by ababaian — CHANGES_REQUESTED** [review]
Cool :) A few minor changes to clean up formatting.

**Comment by joackmak** [comment]
Thank you so much for the feedbacks. I have pushed the changes. Hopefully, they will be reflected. I will appreciate any feedback on what to improve next.

Regards,
Makuach Joack

On Dec 5, 2024, at 3:56 PM, Artem Babaian ***@***.***> wrote:

﻿
You don't often get email from ***@***.*** Learn why this is important<https://aka.ms/LearnAboutSenderIdentification>

@ababaian requested changes on this pull request.

Cool :) A few minor changes to clean up formatting.

________________________________

In virus/Arcanarumenalis.md<https://github.com/ababaian/VIRUSxDISCVRY/pull/64#discussion_r1872062381>:

> @@ -0,0 +1,144 @@
+# Unearthing Arcana ruminalis: A Novel Virus Shaping the Bovine Gut Microbiome in Alberta Ecosystems"
+written by: [Makuach Joack](https://github.com/joackmak *optional link*)


Fix link to your account. Remove the optional linnk text.

________________________________

In virus/Arcanarumenalis.md<https://github.com/ababaian/VIRUSxDISCVRY/pull/64#discussion_r1872064811>:

> +Etiology: The genus name Arcana (Latin for "mystery" or "secret") symbolizes the virus's enigmatic nature and its novel presence in diverse ecosystems. The species name ruminalis highlights its ecological niche in the bovine rumen, where it consistently interacts with the microbiome, playing a potentially significant role in microbial dynamics and host adaptation [6]}*
+
+image of my obelisk
+
+## Abstract: Characterization of Arcana ruminalis in the Bovine Gut Ecosystem
+Viruses and viroid-like entities are integral to understanding microbial ecology and evolution. They interact dynamically with their environments, influencing host systems and microbial community structure. Within this context, we investigated Arcana ruminalis, a novel circular RNA virus-like entity classified in the "obelisk" group, which is consistently detected in bovine rumen microbiomes, particularly in Alberta, Canada.
+
+Through computational and structural analyses, we identified Arcana ruminalis as a viroid-like entity with a compact 785-nucleotide circular RNA genome containing six predicted open reading frames (ORFs). Notably, Arcana ruminalis lacks a conserved RNA-dependent RNA polymerase (RdRp), a hallmark of many RNA viruses, suggesting reliance on host or microbial enzymes for replication. Predicted rod-like secondary structures and the absence of conserved domains further distinguish it from known viral families.
+
+Here, we show that Arcana ruminalis exhibits a strong ecological association with anaerobic environments like the rumen, where it may play a role in shaping microbial communities or interacting with host metabolic processes. ORFs within its genome display weak homologies to proteins from bacteria, fungi, and cyanobacteria, indicating possible horizontal gene transfer or convergent evolution. These findings challenge conventional definitions of RNA viruses and highlight the potential of Arcana ruminalis as a key player in microbial dynamics.
+
+Our results contribute to a broader understanding of novel RNA-based entities, shedding light on their ecological roles and evolutionary pathways. This work underscores the importance of characterizing unconventional genomes to redefine the boundaries of viral taxonomy and their functional significance within host ecosystems.
+
+## Results
+
+### Ecology is defined as the interaction between the organisms and their environment, it may be biotic (consider other organisms as living components) and abiotic (the non-living homeostasis, such as climate, soil, and water). Computational analysis of genomic and environmental data sheds light on relationships between viral organisms and their environments in the context of digital ecology. Digital ecology utilizes bioinformatics tools to investigate the host−virus interaction of viruses like the Obelisk in multiple host species and environmental conditions in multiple ecosystems using the data acquired from sequencing technologies and other environmental datasets.


This is title, it won't look good if the paragraph is all in title-space. Make the title descriptive of your primary conclusion for that section.

________________________________

In virus/Arcanarumenalis.md<https://github.com/ababaian/VIRUSxDISCVRY/pull/64#discussion_r1872065610>:

> +Arcana ruminalis has been described in the study by Fuyong Li et al. (2019) -Rumen microbiomes and their associations with feed efficiency in various beef cattle breeds, where the virus was identified through high-throughput sequencing of bovine gut metagenome samples,Bioproject: PRJNA448333, University of Alberta. The publication aims to identify differences between functional potentials and activities of the same rumen microbiomes and to evaluate the effects of host breed and feed efficiency on the rumen microbiome. It was also described in the studies by Wang Y et la (2024) and Sun HZ et la (2021) at the Functional Genomics and Microbiology, College of Animal Sciences, Zhejiang University, Bioproject: PRJNA480220. Both of these studies were looking at optimized methods to identify circular RNAs among the cattle tissues and identifying factors regulating fatty acid composition in cattle respectively.
+
+Arcana ruminalis was identified in several SRA runs, including SRR8416055,  SRR8416060, and SRR7498097, all of which are either associated with the BioProjects PRJNA448333 or PRJNA480220. These datasets include metagenomic sequencing data from tissue samples of beef cattle (mostly rumen) all raised in Albarta,Canada. The sequencing technologies used in these datasets include Illumina sequencing especially  Illumina HiSeq 2500 and Illumina HiSeq 4000, which provide high-resolution data for viral detection and identification. These runs focus on studying viral diversity in ecosystems such as the gut of the cattle.
+
+
+Upon analyzing the datasets containing my index run and its associated clusters, several common factors emerge. To begin with, the obelisk is consistently detected in the bovine gut ecosystems. Moreover, the bovine species- Bos taurus, where it is persistently found were raised in Albarta. In comparsion with broader ecological aspect of the other datasets, there seems to be a world-wide destribution of the obelisk detection though the detection was skewed toward the countries with more technological will. Interestingly, when I grouped the obelisk_sraRunInfo data by organism and count the number of times an obelisk was detected in an SRA run from that organism, the top hits in which the obelisk was consistently detected in are soil, and the guts of many mammals including bovine, humans and Mucaca mulata.  Observation of the Arcana ruminalis in these environment and organisms stimulated several questions to pursue. What is the evolutionary origin of this Obelisk, Arcana ruminalis?. Does this it interact with a specific host? Does it possess any unique open reading frames (ORFs) that could encode novel proteins? What is its geographical restriction? Does it encode non-coding RNAs? Is there a possibility of horizontal transfer?
+
+Hypothesis:
+The consistent detection of the Arcana ruminalis in bovine gut ecosystems, particularly in Alberta, Canada, suggests a potential association between the Obelisk and the bovine gut microbiome. This observation leads to the hypothesis that Arcana ruminalis may interact with specific host species, particularly bovine species, and may play a role in the gut microbiome's composition and function.
+
+To address the hypothesis that Arcana ruminalis interacts with specific host species, particularly bovine species, I queried PebbleScout[4] using the nucleotide sequences of Arcana ruminalis. The analysis revealed that the first 100 cases consistently exhibited high raw scores, PBS scores, and percentage coverage across all SRA datasets derived from rumen microbiomes. In contrast, detection in datasets from non-rumen sources was significantly lower.
+
+This strong signal in rumen-specific data likely support the hypothesis that Arcana ruminalis is closely associated with bovine gut ecosystems, specifically the rumen. Its absence or low prevalence in other datasets reinforces its potential specialization or ecological dependence on the bovine gut microbiome. These findings hint to suggest that Arcana ruminalis may indeed play a functional role in the microbial dynamics of the rumen, warranting further investigation into its interactions and contributions within this environment.
+
+
+### Virus Genome {Q3}


Amend out the "Default Titles" from the document. Your descriptive titles are much better throughout.

________________________________

In virus/Arcanarumenalis.md<https://github.com/ababaian/VIRUSxDISCVRY/pull/64#discussion_r1872066573>:

> +
+Hypothesis:
+The consistent detection of the Arcana ruminalis in bovine gut ecosystems, particularly in Alberta, Canada, suggests a potential association between the Obelisk and the bovine gut microbiome. This observation leads to the hypothesis that Arcana ruminalis may interact with specific host species, particularly bovine species, and may play a role in the gut microbiome's composition and function.
+
+To address the hypothesis that Arcana ruminalis interacts with specific host species, particularly bovine species, I queried PebbleScout[4] using the nucleotide sequences of Arcana ruminalis. The analysis revealed that the first 100 cases consistently exhibited high raw scores, PBS scores, and percentage coverage across all SRA datasets derived from rumen microbiomes. In contrast, detection in datasets from non-rumen sources was significantly lower.
+
+This strong signal in rumen-specific data likely support the hypothesis that Arcana ruminalis is closely associated with bovine gut ecosystems, specifically the rumen. Its absence or low prevalence in other datasets reinforces its potential specialization or ecological dependence on the bovine gut microbiome. These findings hint to suggest that Arcana ruminalis may indeed play a functional role in the microbial dynamics of the rumen, warranting further investigation into its interactions and contributions within this environment.
+
+
+### Virus Genome {Q3}
+
+To annotate the genome of Arcana ruminalis, I began by running the sequence of the centroid through NCBI ORFfinder  to identify all potential open reading frames (ORFs). Each ORF was subsequently analyzed through BLASTp searches to determine homology and functional annotations for the predicted proteins. Using the results, I compiled a comprehensive genomic dataframe that integrated the ORF details with their corresponding BLASTp annotations. Finally, leveraging R, I created a detailed genome map to visually represent the annotated features and provide a clear overview of the genomic organization.
+A. ruminalis Genome Map
+
+
+Figure 1 : Annotation of Arcana rumenalis genome.


Figure Legend?

**Review by ababaian — APPROVED** [review]

**Pull request merged**